tests/daemon/test-daemon.c

```diff
@@ -64,6 +64,9 @@
 
 #define TIMEOUT_SEC 5
 
+#define INTERVAL_SEC 3600
+#define INTERVAL_USEC (INTERVAL_SEC + G_USEC_PER_SEC)
```

Comment:
Should this be `INTERVAL_SEC * G_USEC_PER_SEC` rather than `+`?

Reply:
**Great** catch. I found the other confusion above and it's fixed in my working copy, but then had some really baffling failures in the test suite. I stared at the logs for hours before giving up!